tests/ui/openid_connect_mock.rs

```diff
@@ -106,14 +161,34 @@ type KnownUsers = HashMap<KnownUserId, KnownUser>;
 const DEFAULT_TOKEN_DURATION_SECS: u32 = 3600;
 static MOCK_OPENID_CONNECT_SERVER_RUNNING_FLAG: AtomicBool = AtomicBool::new(false);
 
-pub async fn start() -> Option<task::JoinHandle<()>> {
+#[tokio::main]
+pub async fn main() {
+    // Log to stdout.
+    #[cfg(feature = "multi-user")]
+    let _logger = fern::Dispatch::new().format(move |out, message, record| {
+            out.finish(format_args!(
+                "{} [{}] {}",
+                chrono::Local::now().format("%Y-%m-%d %H:%M:%S"),
+                record.level(),
+                message
+            ))
+    })
+    .level(log::LevelFilter::Trace)
+    .chain(std::io::stdout())
+    .apply()
+    .map_err(|e| format!("Failed to init stderr logging: {}", e));
+    
+    start(2500).await;
+}
+
+pub async fn start(delay_secs: u64) -> Option<task::JoinHandle<()>> {
```

Comment:
What was the motivation for making this configurable, especially since the caller uses a hard-coded value?

Reply:
My take at the lowest-effort-way of making the file openid\_connect\_mock.rs into a stand-alone server, that would run for the specified amount of seconds (See the main() function).